.github/workflows/validate.yaml

```diff
@@ -0,0 +1,112 @@
+name: VALIDATE
```

Comment:
Will the job run in parallel or sequentially when they are all in one file?

Reply:
Everything runs in parallel.